src/vs/workbench/browser/parts/titlebar/titlebarPart.ts

```diff
@@ -326,6 +327,7 @@ export class TitlebarPart extends Part implements ITitleService {
 			activeFolderMedium,
 			activeFolderLong,
 			rootName,
+			rootNameShort,
```

Comment:
A new variable needs to be added to here as well:

https://github.com/microsoft/vscode/blob/4a130c40ed876644ed8af2943809d08221375408/extensions/configuration-editing/src/settingsDocumentHelper.ts#L81

and also here:

https://github.com/microsoft/vscode/blob/1128bdab1bcf92f643afee2af1d6c055c8475081/src/vs/workbench/browser/workbench.contribution.ts#L487